model/project.rb

```diff
@@ -37,6 +37,21 @@ def has_valid_payment_method?
     !!billing_info&.payment_methods&.any?
   end
 
+  def default_location
```

Comment:
Note: Added `default_location` to project as it seams reasonable to let the project decide the default location and in the future there could be extensions, e.g., customers could pick their favorite location as default. But feel free to object if you feel this logic shouldn't go to project.